components/homePage/MessageBox.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+    Box,
+    Button,
+    Typography,
+    Modal,
+    DialogContent,
+  } from "@mui/material";
+import modalImage from "../../public/images/homePage/dialogBox/thumbUp.webp";
+import Image from "next/image";
+import { bannerDetails } from "../../data/homePage";
+import * as React from "react";
+
+// modal styles
+const style = {
+    position: "absolute" as "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: { xs: "85%", sm: "60%", md: "35%" },
+    height: "auto",
+    bgcolor: "background.paper",
+    boxShadow: 50,
+    p: 4,
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    borderRadius: "10px",
+  };
+
+  
+//logic to open modal and functionalities
+const MessageBox = () => {
+    const [open, setOpen] = React.useState(false);
```

Comment:
Document your functions

// Function to open the modal

// Function to close the modal